ruma-identity-service-api/src/association/email/create_email_validation_session/v2.rs

```diff
@@ -0,0 +1,51 @@
+//! [POST /_matrix/identity/v2/validate/email/requestToken](https://matrix.org/docs/spec/identity_service/r0.3.0#post-matrix-identity-v2-validate-email-requesttoken)
+
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Create a session for validating an email address.",
+        method: POST,
+        name: "create_email_validation_session",
+        path: "/_matrix/identity/v2/validate/email/requestToken",
+        authentication: AccessToken,
+        rate_limited: false,
+    }
+
+    request: {
+        /// A unique string generated by the client, and used to identify the validation attempt.
+        pub client_secret: &'a str,
```

Comment:
In the future, we may want to use an identifier type for this (and the `sid` response field) . I created  #517 about that. `&'a str` is fine for now.

Reply:
okay :+1: . WIll refactor that once done. I think I'll take on the issue myself one IS API is complete.